pkg/kubelet/nodestatus/setters.go

```diff
@@ -249,6 +279,76 @@ func hasAddressValue(addresses []v1.NodeAddress, addressValue string) bool {
 	return false
 }
 
+func findFqdn(name string, domains []string) string {
+	if len(domains) < 1 {
+		return name
+	}
+
+	// If our hostname already ends with one of the search domains, treat it as a FQDN
+	for _, d := range domains {
+		if strings.HasSuffix(name, d) {
+			return name
+		}
+	}
+
+	// Otherwise, use the first search domain as the conventional DNS domain
+	return name + "." + domains[0]
+}
+
+func findHostIPsViaDNS(name string, preferIPv4 bool) (out []net.IP) {
+
+	// Read resolv.conf to find servers
+	cfg, err := dns.ClientConfigFromFile("/etc/resolv.conf")
+	if err != nil {
+		// Otherwise, try reasonable defaults
```

Comment:
Assuming this function is kept at all, no. If they don't have an `/etc/resolv.conf`, don't do DNS.

Reply:
Not that I care about them, but what about platforms which do not use /etc/resolv.conf?  I assume this is the case, for instance, on Windows.